definitions/private-law/json/FixedLists.json

```diff
@@ -6539,5 +6539,54 @@
     "ListElementCode": "physicalAbuse",
     "ListElement": "Physical abuse",
     "DisplayOrder": 1
+  },
+  {
+    "LiveFrom": "05/12/2023",
+    "ID": "JudgeApprovalDecisionsCourtAdminEnum",
+    "ListElementCode": "sendToAdminToServe",
+    "ListElement": "Send to admin to serve",
+    "DisplayOrder": 1
+  },
+  {
+    "LiveFrom": "05/12/2023",
+    "ID": "JudgeApprovalDecisionsCourtAdminEnum",
+    "ListElementCode": "giveAdminFurtherDirectionsAndServe",
+    "ListElement": "Give admin further directions then serve",
+    "DisplayOrder": 2
+  },
+  {
+    "LiveFrom": "05/12/2023",
+    "ID": "JudgeApprovalDecisionsCourtAdminEnum",
+    "ListElementCode": "editTheOrderAndServe",
+    "ListElement": "Edit the order myself and send to admin to serve",
+    "DisplayOrder": 3
+  },
+  {
+    "LiveFrom": "05/12/2023",
+    "ID": "JudgeApprovalDecisionsSolicitorEnum",
+    "ListElementCode": "sendToAdminToServe",
```

Comment:
These element codes needs to be changed for Admin and solicitor, both have the same element code.

Reply:
As long as Id is unique we can make use of same ListelementCodes